corporate/tests/test_stripe.py

```diff
@@ -965,7 +965,7 @@ def test_upgrade_by_card_to_plus_plan(self, *mocks: Mock) -> None:
             .order_by("id")
         )
         self.assertEqual(
-            audit_log_entries[:3],
+            audit_log_entries[3:6],
```

Comment:
There are more audit log entries now when the default `can_manage_group` is created. This applies to all similar changes across this file.

Reply:
audit log entries were created because we were creating a group with hamlet as the active user in populate_db. In hindsight, simpler thing to do would have been just change that user to someone else in populate_db. I've done that and all audit_log related changes have been reverted.